server/appengine/src/main/java/who/PublicWhoRpcFilter.java

```diff
@@ -0,0 +1,26 @@
+package who;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import present.rpc.LoggingInterceptor;
+import present.rpc.RpcFilter;
+import present.rpc.RpcInterceptorChain;
+
+/**
+ * Configures RPC services for publicly-cacheable APIs.
+ */
+@Singleton public class PublicWhoRpcFilter extends RpcFilter  {
```

Comment:
We don't need this class. For future reference, we need just one RpcFilter class for the whole app. You can call service() multiple times.